static/app/components/panels/panelHeader.tsx

```diff
@@ -41,3 +42,21 @@ const PanelHeader = styled('div')<Props>`
 `;
 
 export default PanelHeader;
+
+const ClickablePanelHeader = styled(PanelHeader)`
+  user-select: none;
```

Comment:
Why do we want to disable text selection in the header? I can see a use case where the user would wanna select and copy the header text